packages/vite/src/node/server/index.ts

```diff
@@ -728,10 +727,13 @@ export async function _createServer(
 
   const publicFiles = await initPublicFilesPromise
 
-  const onHMRUpdate = async (file: string, configOnly: boolean) => {
+  const onHMRUpdate = async (
+    type: 'create' | 'delete' | 'update',
```

Comment:
We have an opportunity to define new names for the new `hotUpdate` API, so maybe we could discuss what are the best ones here already. I'm good with `create`, `delete`, `update` (Parcel Watcher and others use that). I imagine it is better than `add`, `unlink`, `change` names from Chokidar? But fs.watch in node uses `change` too, no?
This is a internal type for now, so I'm good with you merging it as is and we can discuss once we need to expose it.

Reply:
I aligned these names with the [`watchChange` hook](https://rollupjs.org/plugin-development/#watchchange).

Reply:
Ah, perfect 👍🏼